web/src/modules/administration/router/index.ts

```diff
@@ -22,6 +22,14 @@ const routes: RouteRecordRaw[] = [
       requireSystemAdmin: true,
     },
   },
+  {
+    path: "administration/injest-data",
+    component: () =>
+      import("@/modules/administration/modules/injest-data/views/DataIngestionList.vue"),
```

Comment:
Always add a component name to components for more stable routing.
Note route names are global, so I usually include the full file path.
e.g.
- `name: "administration/modules/injest-data/views/DataIngestionList"`
(assuming you won't get conflicts at the module level ...)

I usually flatten these names, and avoid using the "module" pattern, as I feel it is unwarranted for this scale of project. Usually once it _is_ needed, you would be better off forking the entire module out into it's own project.